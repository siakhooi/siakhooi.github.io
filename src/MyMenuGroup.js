import React from 'react'
import PropTypes from 'prop-types'

MyMenuGroup.propTypes = {
  children: PropTypes.object
}

function MyMenuGroup (props) {
  return (
    <div className="MyMenuGroup">{props.children}</div>
  )
}
export default MyMenuGroup
