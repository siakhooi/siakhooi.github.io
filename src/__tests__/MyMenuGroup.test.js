import React from 'react'
import renderer from 'react-test-renderer'
import MyMenuGroup from '../MyMenuGroup'

it('MyMenuGroup/1', () => {
  const component =
    renderer.create(
      <MyMenuGroup><div>ABC</div></MyMenuGroup>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
