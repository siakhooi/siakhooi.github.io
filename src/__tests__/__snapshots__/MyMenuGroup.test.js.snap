// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMenuGroup/1 1`] = `
<div
  className="MyMenuGroup"
>
  <div>
    ABC
  </div>
</div>
`;
